### ISSUE TITLE
IE: Img-element not possible to create

### ISSUE THREAD

**Issue by Detzler** [trevnorris/jqml#2]
Hi,

in IE <= 8 it's not possible to create an img-element.

Trying to append a document fragment to a `IMG` fails:

``` javascript
if (selector) {
    selector.appendChild(fragment);
    return selector;
};
```

It happens even no childs are set:

``` javascript
$.jqml([
  ['img']
]);
```

I guess you have to check either the fragment is empty or not (fragment.childNodes.length) and put the whole into a try/catch-block if childs are supplied to a img-element corresponding to:

``` javascript
$.jqml([
  ['img', [
    ['someChid']
  ]]
]);
```

**Comment by trevnorris** [comment]
I have been able to reproduce the error you are seeing. Though it's only happening in the jQuery package. I'll spend some time locating why this is happening.

**Comment by trevnorris** [comment]
Fixed in 467c5d6
